draft-ietf-webtrans-http2.md

```diff
@@ -268,6 +275,10 @@ contained within HTTP/2 DATA frames on a single HTTP/2 stream, this limit is
 the only mechanism for an endpoint to limit the number of WebTransport streams
 that its peer can open on a session.
 
+An endpoint that wishes to reduce the value of WT_MAX_STREAMS to a value that is
+below the current number of open streams can either close streams that exceed
+the new value or allow those streams to complete.
```

Comment:
I am really confused by this paragraph. I thought WT_MAX_STREAMS only goes up?

Reply:
Ah, sorry, the whole reason we added that clarification :)

Reply:
Removed in [09df5c5](https://github.com/ietf-wg-webtrans/draft-ietf-webtrans-http2/pull/93/commits/09df5c59a1ea251698ca1b072d7b976305a9aed0)